pkg/cli/cvd.go

```diff
@@ -412,3 +450,67 @@ func verifyCVDHostPackageTar(dir string) error {
 	}
 	return nil
 }
+
+type LocalArtifactsSrcs struct {
+	Images            map[string]string
+	Bootloader        string
+	CVDHostPackageTar string
+}
+
+func (s *LocalArtifactsSrcs) validate() error {
+	if s.Bootloader == "" {
+		return errors.New("missing bootloader source")
+	}
+	if s.CVDHostPackageTar == "" {
+		return errors.New("missing cvd host package tar source")
+	}
+
+	// TODO: Make it configurable.
+	var allowedImageKeys = []string{
```

Comment:
How about using `map[string]bool`, not to introduce `contains` function?

Reply:
I see, go 1.21 introduced the [`slices` package](https://pkg.go.dev/slices) which have a `Contain` implementation, look forward to upgrade to repo to go 1.21 soon.

Reply:
Ah, your idea sounds better, if golang 1.21 supports that. Could you just leave a TODO comment in this PR then?

Reply:
yes a TODO comment was needed, however we are no longer doing this checking after changing the sources values to flags.